### DISCUSSION TITLE
Description generation do not work properly for none-English post

### DISCUSSION THREAD

**Discussion by DayuanJiang** [quarto-dev/quarto-cli#5369] [Q&A]
My first paragraph is quite lengthy in my post, but when the description is generated on the index page, it only displays the first few words. How can I control the behavior of the description generation?
My blog site is here: [blog](https://www.jiang.jp/)

**post page**
![post page](https://res.cloudinary.com/jiang/image/upload/v1682935832/Untitled.png)
**index page**
![index page](https://res.cloudinary.com/jiang/image/upload/v1682935832/Untitled_1.png)

**Comment by mcanouil** [comment]
Thanks for the report, could you share a small reproducible blog post which can help us reproduce this? See <https://quarto.org/bug-reports.html>.

**Comment by DayuanJiang** [comment]
@mcanouil 

Here is the post:
```md
---
title: "InPars V2 論文解読"
date: "2023-05-01"
---

InPars V2論文では、Query生成に使用するLLMがGPT3からオープンソースのGPT-J(6B)に変更され、生成したQueryのフィルタリング方法がLog Probabilityからmonot5(3B)をRerankerとして利用する方法に変更された点を挙げている。実験結果としては、V2の精度がV1と比べてわずかに向上したことが報告されている。
```

**Comment by DayuanJiang** [comment]
And if I delete the last 3 Japanese character, it will show the description properly 

```md
---
title: "InPars V2 論文解読"
date: "2023-05-01"
---

InPars V2論文では、Query生成に使用するLLMがGPT3からオープンソースのGPT-J(6B)に変更され、生成したQueryのフィルタリング方法がLog Probabilityからmonot5(3B)をRerankerとして利用する方法に変更された点を挙げている。実験結果としては、V2の精度がV1と比べてわずかに向上したことが報告されて
```

**Reply by mcanouil** [reply]
Thanks, an issue has been opened to track this bug, see #5371 for any follow-up.
